ql/src/java/org/apache/hadoop/hive/ql/ddl/database/create/CreateDatabaseOperation.java

```diff
@@ -44,6 +44,9 @@ public CreateDatabaseOperation(DDLOperationContext context, CreateDatabaseDesc d
 
   @Override
   public int execute() throws HiveException {
+    if (desc.getIfNotExists() && context.getDb().databaseExists(desc.getName())) {
```

Comment:
Should we also follow `table if not exist` check pattern, just do a check in https://github.com/apache/hive/blob/3eb56ecae056d36370ff0ee9a0936c28b431724a/ql/src/java/org/apache/hadoop/hive/ql/ddl/database/create/CreateDatabaseAnalyzer.java#L47-L50

Reply:
Addressed.